services/frontend/src/scenes/Profile/Profile.scss

```diff
@@ -1,54 +1,69 @@
 @import "styles/_colors.scss";
 @import "styles/_dimens.scss";
+@import "styles/_media_queries.scss";
 @import "styles/_mixins.scss";
 
-.profile {
-    align-items: flex-start;
+.profile-content {
     display: flex;
-    flex-direction: row;
-    height: calc(100% - #{$navbar-height});
-    justify-content: space-between;
-    padding: 0;
+    flex: 1;
+    flex-direction: column;
+    padding: $default-spacing*2;
 
-    .profile-content {
-        align-items: center;
-        display: flex;
-        flex-direction: column;
-        height: 100%;
-        overflow: auto;
-        padding: $default-spacing;
-
-        .profile-content-title {
-            color: $color-neutral-6;
-        }
+    .profile-content-title {
+        color: $color-neutral-6;
+    }
 
-        .profile-content-card {
-            margin: 16px $default-spacing*5 $default-spacing 0px;
-            padding: $default-spacing;
-        }
+    .profile-card:first-child {
+        margin-top:$default-spacing;

```

Comment:
Missing space between the property and value...

Reply:
Added the space.